### PR TITLE
change versions for dependencies

### PR DESCRIPTION
**Pull request by nturgut**
making the versions more flexible for packets having different versions on different repositories.

### CODE REVIEW

File: packages/web_drivers/pubspec.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ environment:
 
 dev_dependencies:
```
Comment (yjbanov) [inline packages/web_drivers/pubspec.yaml]: I haven't noticed this before, but I think this should be `dependencies`. `dev_dependencies` are only those used for testing.

File: packages/web_drivers/pubspec.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ environment:
...
   args: 1.5.2
-  http: 0.12.0+4
+  http: ^0.12.0
   path: 1.6.4
-  test: 1.14.2
+  test: ^1.6.5
   yaml: 2.2.0
```
Comment (yjbanov) [inline packages/web_drivers/pubspec.yaml]: We should use `^` for all dependencies.

File: packages/web_drivers/pubspec.yaml (hunk 1 above)
Comment (nturgut) [reply packages/web_drivers/pubspec.yaml]: ah, I thought dev_dependencies for all dev related stuff. Thanks I will change it.

File: packages/web_drivers/pubspec.yaml (hunk 2 above)
Comment (nturgut) [reply packages/web_drivers/pubspec.yaml]: done. thanks!

File: packages/web_drivers/pubspec.yaml (hunk 1 above)
Comment (nturgut) [reply packages/web_drivers/pubspec.yaml]: done.

**Pull request merged**